app/policies/application_policy.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class ApplicationPolicy
+  attr_reader :current_user, :model
+
+  def initialize(current_user, model)
+    raise Pundit::NotAuthorizedError, 'must be logged in' unless current_user
+
+    @current_user = current_user
+    @model = model
+  end
+
+  protected
+
+  def destroy?
+    raise NotImplementedError, 'define method :destroy?'
+  end
```

Comment:
just remove for now
will explain later what is it

Reply:
done